src/web/templates/_batch_load_command_labels.html

```diff
@@ -11,6 +11,17 @@
    const userLanguage = "{{ user|language_preference }}"
    const languages = userLanguage === "en" ? "en" : `${userLanguage}|en`
 
+   function lexemeLabel(id, prop){
+     if (userLanguage in prop) {
+       return `<i>${userLanguage}:</i> ${prop?.[userLanguage]?.value}`
```

Comment:
Avoid these if/else chains if possible:

```javascript
const languages = Object.keys(prop)
const lang = languages.find(language => [userLanguage,  "en"].includes(language)) || languages[0]
return <i>${lang}:</i> ${prop[lang].value}` // remove the "?": You know the key exists, so no need to check for null 
```

Reply:
Also, this function doesn't seem to be doing anything with the `id` parameter...